tests/t30-operator.c

```diff
@@ -2,20 +2,22 @@
 #include <ceed.h>
```

Comment:
This file is the best summary of the proposed changes to q-functions and operators.